hw/ip/flash_ctrl/dv/flash_ctrl_sim_cfg.hjson

```diff
@@ -82,5 +87,9 @@
       name: smoke_hw
       tests: ["flash_ctrl_smoke_hw"]
     }
+    {
+      name: sw_op
+      tests: ["flash_ctrl_sw_op"]
+    }
```

Comment:
we don't need to change a regression target for each test. perhaps we should remove 'smoke_hw` as well.

Reply:
@TimEwins @nikola-miladinovic 
I think we should remove these 2 regression targets, since each of them only has one test. We could just run `dvsim.py -i *sw_op`.

Since the CI has passed, I will merge this PR and let you guys to clean these up in the next PR.